dbinspector/src/main/java/im/dino/dbinspector/fragments/TableListFragment.java

```diff
@@ -205,7 +206,8 @@ private void shareDatabase() {
         Intent intent = new Intent(Intent.ACTION_SEND);
         intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
         intent.setType("application/octet-stream");
-        Uri uri = Uri.parse(String.format("content://%s%s", getString(R.string.dbinspector_authority), database.getAbsolutePath()));
+        Uri uri = Uri.parse(String.format("content://%s%s%s", BuildConfig.AUTHORITY_PREFIX,
```

Comment:
I tested this code. Context#getApplicationInfo().packageName actually returns the same value as applicationId.

I added flavors with applicationIdSuffix and this code worked. Maybe you can think of use case where it won't work?